.github/CODEOWNERS

```
@@ -2,4 +2,4 @@
 # This file controls who is tagged for review for any given pull request.
 
 # For anything not explicitly taken by someone else:
-* @honeycombio/integrations-team @nathanleclaire
\ No newline at end of file
+* @honeycombio/telemetry-team @nathanleclaire
```

Nathan is no longer with Honeycomb.

```suggestion
* @honeycombio/telemetry-team 
```